job-descriptions/software-engineer-manager.md

```diff
@@ -0,0 +1,75 @@
+- **Openness**. We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://docs.sourcegraph.com/dev/releases). Check out [our product roadmap](https://docs.sourcegraph.com/dev/roadmap) to see what we are working on next. We also hold [regular retrospectives](https://docs.sourcegraph.com/dev/retrospectives) to learn how we can work better as a team.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+You will build and run one of the engineering teams at Sourcegraph. Here are our current teams and their missions:
+
+- Code navigation
+  - Provide code intelligence (e.g. hover tooltips, go to definition, find references) across repositories and in many languages.
+  - Integrate Sourcegraph features into existing code hosts (e.g. GitHub, Bitbucket, GitLab, Phabricator) via direct integrations and our browser extensions.
+  - Design an extension API that allows anyone to enhance their code browsing experience in Sourcegraph by surfacing relevant contextual information.
+- Code search & backend
+  - Provide a code search experience that is friendly to new users and powerful in the hands of a veteran user.
+  - Scale code search to return results from tens of thousands of repositories and massive monorepos in under a second.
+- Core services
+  - Build and maintain core backend systems that are critical for the rest of the product.
+  - Ensure that our backend architecture scales to support the needs of our product.
+- Distribution
+  - Guide customers to seamlessly deploy, configure, and upgrade Sourcegraph on-premise in a variety of environments.
+  - Automate release testing and continuous deployments to our test clusters and sourcegraph.com while continuously monitoring important metrics and maintaining zero downtime.
+
+#### Your responsibilities
+
+You will:
+
+- Collaborate with the product team and the head of engineering to define and prioritize the work that the team owns.
+- Communicate the goals of the team and the progress toward those goals.
+- Build relationships with the team through 1:1s and team events.
+- Provide career development advice to engineers to help them grow.
+- Hire to increase capacity or fill missing skills on the team.
+
+#### Your qualifications
+
+- You have managed a team before (preferrably at a small startup) and have experience with the resposibilities listed above.
+- You have enough experience as a software engineer to empathize with the needs of the team and our customers.
```

Comment:
specific level or software engineer in general? Would be a guide to how technical you want the role to be